app/scripts/views/NodeViews/StringView.js

```diff
@@ -1,46 +1,54 @@
 /**
  * Created by Masha on 12/8/2014.
  */
-define(['backbone', 'underscore', 'jquery', 'BaseNodeView'],
-    function(Backbone, _, $, BaseNodeView) {
+define(['backbone', 'underscore', 'jquery', 'BaseNodeView', 'StringBase'],
+    function (Backbone, _, $, BaseNodeView, StringBase) {
 
-        function extraChanged () {
-            var ex = this.model.get('extra');
+    return BaseNodeView.extend(StringBase).extend({
 
-            this.syncUI(ex.value);
-            this.model.trigger('updateRunner');
-            this.model.workspace.trigger('requestRun');
-        }
+        template: _.template($('#node-string-template').html()),
 
-        function focusInput (e) {
-            this.selectable = false;
-            this.model.set('selected', false);
-            e.stopPropagation();
-        }
+        events: _.extend(BaseNodeView.prototype.events, {
```

Comment:
No, It shouldn't. StringBase is not backbone view, it is just a container with common functions. StringView and widget/StringView has different base classes with different implementation, so that is why we do: 
BaseNodeView.extend(StringBase).extend({... 
and
BaseWidgetView.extend(StringBase).extend({...